src/components/dialog/examples/dialog-closing-actions.tsx

```diff
@@ -2,7 +2,36 @@ import { Component, h, State } from '@stencil/core';
 
 /**
  * Custom closing actions
+ *
+ * Action buttons in dialogs can be used to add a clear visual indication for
+ * the sighted users to realize that the dialog can be closed by pressing
+ * a button as well.
+ * This may sometimes be considered an unnecessary usage of action buttons for
+ * sighted users. Because majority of them users know that clicking or tapping
+ * outside the dialog closes it.
```

Comment:
I think this section is a little incoherent

> … can be closed by pressing a button as well.

"As well" as what? It sounds like I'm meant to have read some missing section before this.

The whole paragraph seems to assume that the reader, and all users, see clicking outside a dialog as the _primary_ way to close it, and that having a close-button is some new invention, the purpose of which needs explanation. At least to my ears, that makes this paragraph sound quite weird.

Reply:
@Kiarokh I'd like this paragraph to get a touch-up, but apart from that I'm ready to approve.

Reply:
Good point! What about this: 

to realize that the dialog can be closed by pressing a button as well as the conventional closing way, which is clicking outside the dialog.

Reply:
Yeah, that's better at least. It's still a little weird to me to consider clicking outside the dialog the _main_ way of dismissing it, but sure 👍

Reply:
Will you add a fixup @Kiarokh? I'd love to get this merged 🙂

Reply:
I'm going to approve and merge this. You can always update the text later 🙂